jobs/JGLOBAL_LAND_ANALYSIS_PREP

```diff
@@ -0,0 +1,39 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+export WIPE_DATA="NO"
+export DATA=${DATA:-${DATAROOT}/${RUN}landanl_${cyc}}
+source "${HOMEgfs}/ush/jjob_header.sh" -e "landanlprep" -c "base landanl landanlprep"
+
+##############################################
+# Set variables used in the script
+##############################################
+
+
+##############################################
+# Begin JOB SPECIFIC work
+##############################################
+# Generate COM variables from templates
+YMD=${PDY} HH=${cyc} generate_com -rx COM_OBS
+
+
+###############################################################
+# Run relevant script
+
+EXSCRIPT=${GDASLANDPREPSH:-${HOMEgfs}/scripts/exglobal_land_analysis_prep.sh}
```

Comment:
I would prefer to write all exscripts in python, similar to the aerosol and atmos scripts.

Reply:
This is on the schedule after I move the `apply_jedi_increments` into the `gdaslandanlfinal` job.